Common/src/Common/ArmoniKPayload.cs

```diff
@@ -73,6 +83,20 @@ private static string Base64ToString(string base64)
     var c = Convert.FromBase64String(base64);
     return Encoding.ASCII.GetString(c);
   }
+
+  private void freePayload()
+  {
+    if (ClientPayload is null)
+    {
+      return;
+    }
+
+    Array.Clear(ClientPayload,
```

Comment:
This is not useful as the array reference is set to null just after. BTW, setting a bytearray to all zero does nothing except writing memory.